lessons/babel.md

```diff
@@ -1 +1,130 @@
-babel.md
\ No newline at end of file
+# ModernJS - Babel
+## Welcome to Modern JS
+[Intro to the 4 weeks]
+
+## Babel
+In this lesson we'll be learning about Babel which allows us to write modern javascript without worrying about it!
+
+We'll go through briefly what Babel does, why we use it and how to set it up in a project.
+
+When opening the project and looking around, if you haven't looked at JS for a while there may be some newer syntax you don't understand. It's not just you; some older browsers won't understand this either, which is why we want Babel.
+### What is Babel?
+- It is a 'transpiler', converting Javascript to Javascript
```

Comment:
[transpiler](https://en.wikipedia.org/wiki/Source-to-source_compiler)

Reply:
I agree, also, I'd say the quote marks make it seem like it's not true, I'd make it bold.

Reply:
@shaundillon @svillafe All good stuff - keep it coming!